cli/cyclus.cc

```diff
@@ -350,9 +350,7 @@ int EarlyExitArgs(const ArgInfo& ai) {
     std::cout << Env::nuc_data() << "\n";
     return 0;
   } else if (ai.vm.count("schema")) {
-    std::stringstream f;
-    LoadStringstreamFromFile(f, ai.schema_path);
-    std::cout << f.str() << "\n";
+    std::cout << cyclus::BuildMasterSchema(ai.schema_path) << "\n";
```

Comment:
Coming from my lack of C++ knowledge, what does this change do?

Reply:
I think this is a more concise way to print the schema. Is that correct?

Reply:
The old code just read in the schema template and printed it out, without making the necessary substitutions.  This new way calls the `BuildMasterSchema` function that queries the available archetypes and performs the appropriate substitutions in the schema template.

Reply:
That is very helpful to have.